Packs/ForcepointSecurityManagementCenter/Integrations/ForcepointSecurityManagementCenter/ForcepointSecurityManagementCenter_test.py

```diff
@@ -0,0 +1,662 @@
+from smc.api.exceptions import ElementNotFound
+from smc.base.collection import CollectionManager
+
+
+class mock_Rule:
+    def __init__(self):
+        self.name = 'name'
+        self.tag = 'tag'
+        self.sources = mock_GenericRuleEntity()
+        self.destinations = mock_GenericRuleEntity()
+        self.action = mock_GenericRuleEntity()
+        self.services = mock_GenericRuleEntity()
+        self.comment = 'comment'
+
+    def all(self):
+        return [self]
+
+    def create(self, name, sources, destinations, action, comment):
+        return self
+
+    def update(self, **kwargs):
+        return self
+
+    def save(self,):
+        return self
+
+
+class mock_IPList():
+    def __init__(self, name: str, iplist: list, comment: str):
+        self.name = name
+        self.iplist = iplist
+        self.comment = comment
+
+
+class mock_Host:
+    def __init__(self, name: str, address: str, ipv6_address: str, secondary: str, comment: str):
+        self.name = name
+        self.address = address
+        self.ipv6_address = ipv6_address
+        self.secondary = secondary
+        self.comment = comment
+
+
+class mock_Domain:
+    def __init__(self, name: str, comment: str):
+        self.name = name
+        self.comment = comment
+
+
+class mock_FirewallRule:
+
+    def __init__(self):
+        self.fw_ipv4_access_rules = mock_Rule()
+        self.fw_ipv6_access_rules = mock_Rule()
+
+
+class mock_GenericRuleEntity:
+    def __init__(self):
+        self.name = 'name'
+        self.action = 'ALLOW'
+
+    def all(self):
+        return [self]
+
+    def filter(self):
+        return self
+
+    def add_many(self, elements):
+        return self
+
+
+class mock_Policy:
+    def __init__(self, name: str, comment: str):
+        self.name = name
+        self.comment = comment
+        self.fw_ipv4_access_rules = mock_Rule()
+        self.fw_ipv6_access_rules = mock_Rule()
+
+
+class mock_Engine:
+    def __init__(self, name: str, comment: str):
+        self.name = name
+        self.comment = comment
+        self.pending_changes = mock_Changes()
+
+    def objects(self):
+        return mock_GenericRuleEntity()
+
+
+class mock_Changes:
+    def __init__(self):
+        self.element_name = 'element_name'
+        self.modifier = 'modifier'
+        self.changed_on = 'changed_on'
+
+    def all(self):
+        return self
+
+    def approve_all(self):
+        return
+
+
+def test_create_iplist_command(mocker):
+    """
+    Given:
+        - demisto args
+    When:
+        - Calling function create_address_command
+    Then:
+        - Ensure the results holds the expected data
+    """
+
+    args = {
+        'name': 'name',
+        'addresses': '1.1.1.1,8.8.8.8',
+        'comment': 'comment'
+    }
+    mocker.patch.object(IPList, 'create', return_value=mock_IPList(name='name', iplist=['1.1.1.1', '8.8.8.8'], comment='comment'))
```

Comment:
I'd use `side_effect` instead of `return_value`, and in the `mock_IPList` c'tor accept `name`, `iplist` and `comment` as arguments. Then you won't "lose" `args` and actually use it instead of providing the mock values twice.

I won't insist on this, but please keep that in mind for next time.